src/main/java/edu/umich/tl/ApiCallHandler.java

```diff
@@ -20,44 +20,69 @@
 	private String canvasURL=CourseDelete.canvasURL;
 	private String esbURL=CourseDelete.esbURL;
 	private CanvasCallEnum canvasCall;
-	
 	private static String canvasToken=CourseDelete.canvasToken;
 	private static final String PER_PAGE = "per_page=100";
 	private static final String API_VERSION = "/api/v1";
 	
-	public ApiCallHandler(CanvasCallEnum apiCallType) {
-		this.canvasCall=apiCallType;
+	public ApiCallHandler(CanvasCallEnum canvasCall) {
+		this.canvasCall=canvasCall;
 	}
 	
 	public enum CanvasApiEnum{
-		TERM, UNPUBLISHED_COURSE_LIST;
+		TERM, UNPUBLISHED_COURSE_LIST, UNPUBLISHED_COURSE_LIST_PAGINATION_URL;
 	}
 
-	public HttpResponse getApiResponse(CanvasApiEnum canvasApi) {
+	public HttpResponse getApiResponse(CanvasApiEnum canvasApi, String canvasTermIdForSisTermId, String url) {
```

Comment:
`CanvasApiEnum` and `canvasApi` are simultaneously too wordy and ambiguous names.  I suggest naming the enum `RequestTypeEnum` and the argument `requestType`.